{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/settings/base.py

```diff
@@ -1,8 +1,12 @@
 # Django settings for project project.
-from os import environ
-from django.core.exceptions import ImproperlyConfigured
+from environ import Env, Path
 import pathlib
+from django.core.exceptions import ImproperlyConfigured
 
+env = Env()
```

Comment:
What happened to `.env`? I'm not saying it's wrong I'm just wondering. I looked at the source and it does some funky things to trace it back.